Resources/views/FragmentAdmin/edit_one_association_script_fragment.html.twig

```diff
@@ -0,0 +1,51 @@
+{% autoescape false %}
```

Comment:
I think the darkest part of Sonata is exactly this kind of thing. Leaving this for someone else (or yourself) to fix it later is not right IMO. It should be well separated from the start. Generating code for a description language is ok, but generating code for a programming language is really not IMO. See [this comment](https://github.com/sonata-project/SonataDashboardBundle/pull/47#discussion_r66729309) for more information about that.